### ISSUE TITLE
Check for cookie

### ISSUE THREAD

**Issue by RWOverdijk** [balderdashy/sails.io.js#13]
I'm just wondering a couple of things...

* https://github.com/balderdashy/sails.io.js/blob/master/dist/sails.io.js#L657 could this be simple to implement combined with `urlThisScriptWasFetchedFrom`?
* Using `useCORSRouteToGetCookie` it gets the cookie on every page load it seems. Shouldn't it first check if it perhaps already has a cookie, as to prevent useless calls to the server?
* Socket.io is embedded in the file. I personally don't like this. What's the idea behind that? Can this become optional at some point?
* Socket.io is at 1.0 now, will this be implemented at some point?

Sorry for the bulk of questions :)

**Comment by sailsbot** [comment]
Thanks for posting, @RWOverdijk.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
